docs/Users_Guide/reformat_stat_data.rst

```diff
@@ -46,160 +53,634 @@ Plots that are generated from the command line require a YAML configuration file
+
+        Used by the following METplotpy plot types:
+            - line
+            - bar
+            - box
+            - contour
+            - performance diagram
+            - revision box
+            - revision series
+            - Taylor Diagram
+
+     .. dropdown:: SL1L2
+
+         **Scalar L1L2 Partial Sums**
+
+         From MET tools:
+            - point-stat
+            - grid-stat
+
+        Used by the following METplotpy plot types:
+            - line
+            - bar
+            - box
+            - contour
+            - performance diagram
+            - revision box
+            - revision series
+            - Taylor Diagram
+
+     .. dropdown:: VL1L2
+
+         **Vector L1L2 Partial Sums**
+
+         From MET tools:
+            - point-stat
+            - grid-stat
+
+        Used by the following METplotpy plot types:
+            - line
+            - bar
+            - box
+            - contour
+            - performance diagram
+            - revision box
+            - revision series
+            - Taylor Diagram
+
+     .. dropdown:: ECNT
+
+         **Ensemble Continuous Statistics**
+
+         From MET tools:
+            - point-stat
+            - ensemble-stat
+
+        Used by the following METplotpy plot types:
+            - line (for ensemble spread-skill plots)
+
+     .. dropdown:: MCTS
+
+         **Multi-category Contingency Table Statistics**
+
+         From MET tools:
+            - point-stat
+
+        Used by the following METplotpy plot types:
+            - line
+            - bar
+            - box
+            - contour
+            - performance diagram
+            - revision box
+            - revision series
+            - Taylor Diagram
+
+     .. dropdown:: VCNT
+
+         **Vector Continuous Statistics**
+
+         From MET tools:
+            - point-stat
+            - grid-stat
+
+        Used by the following METplotpy plot types:
+            - line
+            - bar
+            - box
+            - contour
+            - performance diagram
+            - revision box
+            - revision series
+            - Taylor Diagram
+
+     .. dropdown:: PCT
+
+         **Contingency Table counts for Probabilistic forecasts**
+
+         From MET tools:
+            - point-stat
+            - grid-stat
+            - ensemble_stat
+
+        Used by the following METplotpy plot types:
+            - ROC diagram (Receiver Operator Curve)
+
+
+     .. dropdown::  RHIST
+
+         **Ranked Histogram Counts**
+
+         From MET tools:
+            - ensemble-stat
+
+        Used by the following METplotpy plot types:
+            - rank histogram (rhist)
+
+     .. dropdown::  TCDIAG (with TCMPR)
+
+         **Tropcial Cyclone Diagnostics**
+         **Tropical Cyclone Matched Pairs**
+
+         From MET tool:
+            - tc-pairs
+
+        Used by the following METplotpy plot types:
+            - TCMPR (Tropical Cyclone Matched Pair) plots:
+
```

Comment:
This extra line does not show up in RTD:
https://metdataio.readthedocs.io/en/feature_348_dmap_reformat/Users_Guide/reformat_stat_data.html
Please ask @lisagoodrich for assistance if needed.